app/views/rooms/amenities.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="row container amenities">
+    <div class="card left-card">
+        <div class="card-body">
+            <%= render "sidebar" %>
+        </div>
+    </div>
+    <div class="card right-card">
+        <div class="card-header">
+            Amenities
+        </div>
+        <div class="card-body">
+            <%= form_for @room do |f| %>
+                <div class="row row-cols-3">
+                    <div class="col mt-3">
+                        <%= check_box_tag(:tv) %>
```

Comment:
* For our form_for, it would be better to use `f.check_box :has_tv` instead. Please apply the same to the other attributes.
* Please use the attributes from the ERD. Replace `:tv` with `:has_tv`